src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item/ProteinsListItem.scss

```diff
@@ -16,3 +16,9 @@
 .transcriptId {
   font-weight: 400;
 }
+
+
+.scrollRef{
+  position: absolute;
+  top: -55px;
+}
```

Comment:
CSS rule `scroll-margin: 55px;` (or `scroll-margin-top: 55px`) applied to the ProteinsListItem div should make this span unnecessary. It currently has problems on Safari, but Safari will catch up sooner or later.

Reply:
That's done